lib/lightning_web/live/job_live/form_component.ex

```diff
@@ -7,7 +7,11 @@ defmodule LightningWeb.JobLive.FormComponent do
   import Phoenix.LiveView,
     only: [assign: 2, assign: 3]
 
-  alias Lightning.{Jobs, AdaptorRegistry, Projects}
+  alias Lightning.{
```

Comment:
this makes the review harder --> doesn't seem related to the PR

Reply:
Well just `mix format` I guess. but I will change that.

Reply:
@taylordowns2000 In this case, and will always (un/fortunately) lean on `mix format`, so almost impossible to avoid this. Our formatting checker might fail the PR checks if we fail to conform.